static/src/stylesheets/module/content/live-blog/navigation.scss

```diff
@@ -42,82 +39,103 @@
 
 .liveblog-navigation__link {
     display: inline-block;
+    position: relative;
+    color: colour(neutral-1);
+    background: transparent;
+    border: 1px solid colour(neutral-4);
+    height: $gs-baseline * 3;
+    line-height: 38px;
+    font-weight: bold;
+    vertical-align: middle;
```

Comment:
Few questions here (sorry!):

Do you need `background: transparent` here?

Secondly, I wonder if it's possible to style this without hardcoding heights and line heights as magic numbers. This would be preferred because it's less brittle and thus less likely to break in the future, e.g. if the text size changes, you would have to remember to change this as well, I think?